backend/grants/admin.py

```diff
@@ -160,28 +160,14 @@ def _check_amounts_are_not_empty(grant: Grant, request):
         )
         return
 
-    if (
-        grant.grant_type
-        in (
-            Grant.ApprovedType.ticket_accommodation,
-            Grant.ApprovedType.ticket_travel_accommodation,
-        )
-        and grant.accommodation_amount is None
-    ):
+    if grant.has_approved_accommodation() and grant.accommodation_amount is None:
```

Comment:
not for this PR, but we have a very similar logic in the Grant save method when it calculates the amounts to set. We could refactor it to use those 2 new methods